### PR TITLE
clis for non-postgres data tile generation

### PR DESCRIPTION
**Pull request by wavemechanics**
### What

Makefile, scripts, and Go programs to generate Census Map data files locally without postgress or an API.

### CODE REVIEW

**Comment by VivianAllen** [comment]
NB on ds-jargon, the thing called 'content.json' here looks like its actually a data tiles defintion file?

File: data-tiles/README.txt [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+		to the named output file.
+		If the process is killed or if it does not exit 0, then the named
+		output file is not touched.
+		The output file will only be updated if the process exits 0.
+
+	atomic-rm.sh
+		This script atomically removes a directory.
+		If a directory remove operation is stopped prematurely, the directory
+		will be left in an incomplete state and Make would not know about it.
+		So this script renames the directory with a .condemned suffix and then
```
Comment (VivianAllen) [inline data-tiles/README.txt:123]: this is great

File: data-tiles/GNUmakefile [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+
+
+# All the raw geojson files to download
+RAW_GEOJSON=$(RAW_LAD) $(RAW_LSOA) $(RAW_MSOA) $(RAW_OA)
+
+#
+# do everything
+#
+.PHONY: all
+all: dirs $(DOG)/.done $(DOT)/.done $(DOB)/.done
```
Comment (VivianAllen) [inline data-tiles/GNUmakefile:121]: probably missing something here, but just FMI how does this target work? it looks like it just makes some directories to me?

Comment (wavemechanics) [reply data-tiles/GNUmakefile:121]: Further on in the makefile, the .done files are targets with their own dependencies.

Comment (VivianAllen) [reply data-tiles/GNUmakefile:121]: ahhh they're dynamically-named targets? sweet!

File: data-tiles/cmd/split-metrics/main.go [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+	var csvs []string
+
+	err := filepath.Walk(dir, func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if info.Mode()&fs.ModeType != 0 {
+			return nil // must be regular file
+		}
+		isdata, err := filepath.Match("*DATA.CSV", info.Name())
```
Comment (VivianAllen) [inline data-tiles/cmd/split-metrics/main.go:65]: future proofing, might be worth having this match string configurable somehow?

Comment (wavemechanics) [reply data-tiles/cmd/split-metrics/main.go:65]: Yes, if future metrics files arrive with different naming conventions or directory structure, we can tweak that function.

**Review by VivianAllen — APPROVED** [review]
great stuff, works well!

**Pull request merged**